BitwardenShared/UI/Auth/VaultUnlock/VaultUnlockProcessor.swift

```diff
@@ -253,7 +253,17 @@ class VaultUnlockProcessor: StateProcessor<
                 message: "Biometrics auth is enabled but key was unable to be found. Disabling biometric unlock."
             ))
             try? await services.authRepository.allowBioMetricUnlock(false)
-            await loadData()
+
+            let hasMasterPassword = try? await services.authRepository.hasMasterPassword()
+            let isPinEnabled = try? await services.authRepository.isPinUnlockAvailable()
+            if hasMasterPassword == false, isPinEnabled == false {
+                // If biometrics is enabled, but the auth key doesn't exist and the user doesn't
+                // have a master password or PIN, log the user out.
+                await logoutUser(userInitiated: false)
+            } else {
+                // Otherwise, refresh the data to remove biometrics as an unlock method.
+                await loadData()
+            }
```

Comment:
🤔 If `hasMasterPassword` is `nil` because of throwing shouldn't we logout the user as well? Given that either there are no accounts or there are no active accounts; so I guess on these cases it makes sense to logout as well.